FND/Places/VirtualPlaces.rdf

```diff
@@ -49,13 +49,36 @@
 		<owl:imports rdf:resource="https://spec.edmcouncil.org/fibo/ontology/FND/Utilities/AnnotationVocabulary/"/>
 		<owl:imports rdf:resource="https://www.omg.org/spec/LCC/Countries/CountryRepresentation/"/>
 		<owl:imports rdf:resource="https://www.omg.org/spec/LCC/Languages/LanguageRepresentation/"/>
-		<owl:versionIRI rdf:resource="https://spec.edmcouncil.org/fibo/ontology/FND/20200201/Places/VirtualPlaces/"/>
+		<owl:versionIRI rdf:resource="https://spec.edmcouncil.org/fibo/ontology/FND/20200301/Places/VirtualPlaces/"/>
 		<skos:changeNote>The http://www.omg.org/spec/EDMC-FIBO/FND/20141101/Places/VirtualPlaces.rdf version of this ontology was modified for the FIBO 2.0 RFC to integrate it with LCC.</skos:changeNote>
-		<skos:changeNote>The https://spec.edmcouncil.org/fibo/ontology/FND/20180801/Places/VirtualPlaces.rdf version of this ontology was modified to eliminate duplication of concepts in LCC.</skos:changeNote>
+		<skos:changeNote>The https://spec.edmcouncil.org/fibo/ontology/FND/20180801/Places/VirtualPlaces.rdf version of this ontology was modified to eliminate duplication of concepts in LCC and email address and telephone number.</skos:changeNote>
 		<skos:changeNote>This ontology was added to Foundations in advance of the December 2014 Long Beach meeting in support of the SEC specification.</skos:changeNote>
 		<fibo-fnd-utl-av:hasMaturityLevel rdf:resource="&fibo-fnd-utl-av;Release"/>
 	</owl:Ontology>
 	
+	<owl:ObjectProperty rdf:about="&fibo-fnd-plc-adr;hasElectronicMailAddress">
```

Comment:
Actually, in Wells Fargo's enterprise glossary it is expanded exactly as provided here.  It is their preferred term, though some systems of record (SORs) certainly abbreviate it.

Reply:
We have a policy not to use acronyms / abbreviations in names, at least of classes and properties, with exceptions around individuals, so I think it should be spelled out properly.

Reply:
I think almost everyone treats "email" as a word, just as people use "bus" instead of "omnibus" and "pram" instead of "perambulator". However if you want to treat it as an abbreviation then at least record "email" as abbreviation using the annotation property.

Reply:
I agree that "email" as an abbreviation should appear here.  I checked out what I believe to be the latest, and I don't see the synonym.   Am I in the wrong branch, or did this not get done?  It seems easy enough, and should be done.

Reply:
@dallemang - it's there, on both the class and on the property.  You'll see it if you look at the file itself.